src/cadence/scripts/getNFTs.cdc

```diff
@@ -1229,3 +1230,39 @@ pub fun getGeniaceNFT(owner: PublicAccount, id: UInt64): NFTData? {
         },
     )
 }
+
+// https://flow-view-source.com/mainnet/account/0xf5b0eb433389ac3f/contract/Collectible
+// https://flow-view-source.com/testnet/account/0x85080f371da20cc1/contract/Collectible
+pub fun getXtinglesNFT(owner: PublicAccount, id: UInt64): NFTData? {
+    let contract = NFTContract(
+        name: "Xtingles",
+        address: 0xf5b0eb433389ac3f,
+        storage_path: "Collectible.CollectionStoragePath",
+        public_path: "Collectible.CollectionPublicPath",
+        public_collection_name: "Collectible.CollectionPublicPath",
+        external_domain: "https://www.xtingles.com/"
+    )
+
+    let col = owner.getCapability(Collectible.CollectionPublicPath)
+        .borrow<&{Collectible.CollectionPublic}>()
+    if col == nil { return nil }
+
+    let nft = col!.borrowCollectible(id: id)
+    if nft == nil { return nil }
+
+    return NFTData(
+        contract: contract,
+        id: nft!.id,
+        uuid: nil,
+        title: nft!.metadata!.name,
+        description: nft!.metadata!.description,
+        external_domain_view_url: nil,
+        media: nil,
```

Comment:
Can we update the media link? I'm guessing its nft!.metadata!.link?

Reply:
Yes. I updated media.